content/response-to-reviewers.md

```diff
@@ -53,6 +53,9 @@ We found this approach allows users to modify source code at a scale that is mor
 What new features will be implemented?
 Do you plan to apply for a grant to further develop manubot? etc.
 
+We discuss this issue in [GH133](https://github.com/greenelab/meta-review/issues/133) and address it in [GH179](https://github.com/greenelab/meta-review/pull/179).
+Specifically, we added a paragraph to the Discussion about proposed enhancements to Manubot.
```

Comment:
`paragraph` could be `sub-section`.  You could also mention the grant that supports Manubot and suggest we'll continue to pursue relevant funding opportunities (if that is the case).

Reply:
Done in https://github.com/greenelab/meta-review/pull/179/commits/28eab4b7bd3adc019f001e06ad64228c74806237